helm-charts/sysdig/templates/daemonset.yaml

```diff
@@ -125,7 +125,7 @@ spec:
           livenessProbe:
             exec:
               command: [ "test", "-e", "/opt/draios/logs/running" ]
-            initialDelaySeconds: 10
+            initialDelaySeconds: {{ .Values.daemonset.livenessProbe.initialDelay }}
```

Comment:
I would use a hard coded default value so if there is nothing in the values file it does not mess up things. 

Example:
```
initialDelaySeconds: {{ .Values.daemonset.livenessProbe.initialDelay | default 10 }}
```

Reply:
We will include in the values.yaml file and document it properly:

https://helm.sh/docs/chart_best_practices/values/#document-valuesyaml

Reply:
My comment was regardless of the documentation (which @sarangard did update).
Having built-in defaults make the charts more resilient: https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function

Reply:
I chose to put the default in values.yaml instead of here. The reason for doing so is all the defaults are listed in values.yaml in one place.